lib/blockchain/chain.js

```diff
@@ -550,6 +550,24 @@ class Chain extends AsyncEmitter {
             100);
         }
 
+        if (this.height >= this.network.goosigStop) {
+          const key = proof.getKey();
+
+          if (!key) {
+            throw new VerifyError(block,
+              'invalid',
+              'bad-airdrop-sanity',
+              100);
+          }
+
+          if (key.isGoo()) {
+            throw new VerifyError(block,
+              'invalid',
+              'bad-airdrop-disabled',
```

Comment:
Should the reason be 'bad-goosig-disabled'?

Reply:
Updated to this